lib/screens/home/profile.dart

```diff
@@ -25,41 +25,14 @@ class _ProfileState extends State<Profile> {
   String handle;
   String error = '';
   String key = '';
-
+  String newHandle;
   DatabaseMethods databaseMethods = new DatabaseMethods();
   QuerySnapshot currentUser;
 
```

Comment:
My intention was to display the user's handle dynamically at the top of profile picture